runtime/extism.h

```diff
@@ -106,3 +106,5 @@ const uint8_t *extism_plugin_output_data(struct ExtismContext *ctx, ExtismPlugin
  * Set log file and level
  */
 bool extism_log_file(const char *filename, const char *log_level);
+
+const uint8_t *extism_version(void);
```

Comment:
This appears to be generated but i'm not sure from where. missing a docstring

Reply:
I think converting this comment to a doc comment should copy the documentation over: https://github.com/extism/extism/pull/30/files#diff-9c18909a3b1c4dcdf57dc4edc707e98ba17728203f7c17d573a5d1de60ba9095R389

Reply:
(It's generated from sdk.rs using cbindgen in the build script)

Reply:
yes, appears i just annotated that comment incorrectly